website/docs/reference/warehouse-setups/dremio-setup.md

```diff
@@ -5,7 +5,7 @@ meta:
   authors: 'Dremio (formerly Fabrice Etanchaud)'
   github_repo: 'dremio/dbt-dremio'
   pypi_package: 'dbt-dremio'
-  min_core_version: 'v1.1.0'
+  min_core_version: 'v1.3.0'
```

Comment:
to me, `min_core_version` means the oldest version of dbt-core with which the adapter is compatible. so i interpret this change to mean that there is no version of dbt-dremio that is compatible with `dbt-core==1.1.0`, only from `1.3.0= and onwards...

Reply:
Gotcha.

We should have min version being v1.1.0 and will update. We do support dbt-core v1.1.0, but do not support any earlier versions. Fabrice's adapter would have be used for those.